pkg/rtc/room.go

```diff
@@ -464,6 +464,11 @@ func (r *Room) RemoveParticipant(identity livekit.ParticipantIdentity, pID livek
 	// send broadcast only if it's not already closed
 	sendUpdates := !p.IsDisconnected()
 
+	// remove all published tracks
+	for _, t := range p.GetPublishedTracks() {
```

Comment:
drive-by. this cleans up tracks so any subscribers would get immediate removal.. instead of having SubscriptionManager keep trying to reconcile until timeout.